gen/aircraft.py

```diff
@@ -1747,7 +1749,9 @@ def build(self) -> MovingPoint:
         orbit = OrbitAction(
             altitude=waypoint.alt,
             pattern=OrbitAction.OrbitPattern.RaceTrack,
-            speed=int(flight_plan.patrol_speed.kph),
+            speed=int(
+                flight_plan.patrol_speed.kph if hasattr(self, "patrol_speed") else 600
```

Comment:
getattr has an optional third argument for a default when the attribute isn't found that does this more idiomatically